### ISSUE TITLE
Link in Dropdown Menu

### ISSUE THREAD

**Issue by Edsol** [Holt59/cakephp3-bootstrap-helpers#159]
[x] bug
 [x] question

CakePHP Version: 3.6

I want to create a dropdown menu with links to use:
```
$operations=[
    $this->Html->link("Test Link1","/link1"),
    $this->Html->link("Test Link2","/link2"),

echo $this->Form->dropdownButton('Operations',$operations,['class'=>'btn btn-info']);
```
![screenshot_20180510_101010](https://user-images.githubusercontent.com/19670024/39859528-6de3d4f2-543a-11e8-95f0-88324a9f47ff.png)

but the url does not appear in the code so there is no link

![immagine](https://user-images.githubusercontent.com/19670024/39859599-a2ca561e-543a-11e8-949b-1ffc441289da.png)

if I copy the same code outside from array like:
`
echo $this->Html->link("Test Link1","/link1");`

it's work perfectly:

![immagine](https://user-images.githubusercontent.com/19670024/39859736-1a46a544-543b-11e8-9db0-c5418ea661b9.png)

Is there something that doesn't work in the dropdown menus or am I wrong to create the code?

**Comment by Holt59** [comment]
Thanks for the feedback, this was effectively a bug. I fixed it in d9ade8bd84b36b2a807f8005fcff55715a8d7ba8, you should be able to update via composer.

**Comment by Edsol** [comment]
thanks a lot for quickly response and fix. Work perfectly now, also using Controller and action of cakephp.
